src/main/resources/project_example.json

```diff
@@ -41,22 +42,22 @@
     {
       "type": "ProjectManager",
       "identity": "https://entitydata.example.org/person/23911",
-      "affiliation": "https://entitydata.example.org/organization/185.13.13.0"
+      "affiliation": [ "https://entitydata.example.org/organization/185.13.13.0" ]
     },
     {
       "type": "Participant",
       "identity": "https://entitydata.example.org/person/4583",
-      "affiliation": "https://entitydata.example.org/organization/185.13.13.0"
+      "affiliation": [ "https://entitydata.example.org/organization/185.13.13.0" ]
     },
     {
       "type": "Participant",
       "identity": "https://entitydata.example.org/person/4514",
-      "affiliation": "https://entitydata.example.org/organization/185.13.13.0"
+      "affiliation": [ "https://entitydata.example.org/organization/185.13.13.0" ]
```

Comment:
(As mentioned on Slack)

Consider attaching the participant type to each org. affiliation, not the participant as such.

Cristin attempted a few years back to simplify the model as shown here (type/role as a singular attribute of the participant) but got blowback from users immediately. Users (i.e. healthcare institutions) required being able to model that participant X represents institution A as project manager and institution B as participant.